codalab/client/local_bundle_client.py

```diff
@@ -12,12 +12,10 @@
 import os
```

Comment:
Since we've gotten rid of the the XMLRPC server, why can't we get rid of the bundle client now?